packages/deploy/README.md

```diff
@@ -78,6 +83,7 @@ There are a number of optional fields depending on what you are deploying, these
 - `salt` - deployments are done using the CREATE2 opcode, you can provide a salt or we can generate one for you if none is supplied
 - `licenseType` - This will be displayed on Etherscan e.g MIT
 - `libraries` - If you contract uses any external libraries they will need to be added here in the format `{ [LibraryName]: LibraryAddress }`
+- `walletId` - The wallet ID in case you wish to override the default global approval process
```

Comment:
maybe some more description mentioning that the wallet will be used as approval process (not sure outsiders will make the connection between approval process and wallet)